prow/cmd/checkconfig/main.go

```diff
@@ -271,7 +276,6 @@ func validate(o options) error {
 		if err != nil {
 			return fmt.Errorf("error loading GitHub client: %w", err)
 		}
-		githubClient.Throttle(3000, 100) // 300 hourly tokens, bursts of 100
```

Comment:
I think this has a zero too much :D

Reply:
Heh. It's this way since 7 Aug 2019 when I made that mistake in https://github.com/kubernetes/test-infra/pull/13627 so maybe let's keep it if nobody complained until now?